store/store.go

```diff
@@ -196,29 +170,6 @@ func (s *DefaultStore) GetSignature(ctx context.Context, height uint64) (*types.
 	return &signature, nil
 }
 
-// SaveExtendedCommit saves extended commit information in Store.
-func (s *DefaultStore) SaveExtendedCommit(ctx context.Context, height uint64, commit *abci.ExtendedCommitInfo) error {
-	bytes, err := commit.Marshal()
-	if err != nil {
-		return fmt.Errorf("failed to marshal Extended Commit: %w", err)
-	}
-	return s.db.Put(ctx, ds.NewKey(getExtendedCommitKey(height)), bytes)
-}
-
-// GetExtendedCommit returns extended commit (commit with vote extensions) for a block at given height.
-func (s *DefaultStore) GetExtendedCommit(ctx context.Context, height uint64) (*abci.ExtendedCommitInfo, error) {
-	bytes, err := s.db.Get(ctx, ds.NewKey(getExtendedCommitKey(height)))
-	if err != nil {
-		return nil, fmt.Errorf("failed to load extended commit data: %w", err)
-	}
-	extendedCommit := new(abci.ExtendedCommitInfo)
-	err = extendedCommit.Unmarshal(bytes)
-	if err != nil {
-		return nil, fmt.Errorf("failed to unmarshal extended commit: %w", err)
-	}
-	return extendedCommit, nil
-}
```

Comment:
aren't we planning to have this for attester system and oracles?

Reply:
I guess this should be a part of ABCI-specific execution code.